### PR TITLE
Release 2.4.0

### PR DESCRIPTION
**Pull request by Guts**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,7 +8,11 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/), and this
 
 ## Unreleased
 
-* Keep files permissions when creating zip file.
+## 2.4.0 - 2022-08-25
+
+* Keep files permissions when creating zip file by @jmkerloch #129
+* update dependencies
+* documentation improvments
```
Comment (Gustry) [inline CHANGELOG.md:15]:
```suggestion
* Update dependencies
* Documentation improvements
```

Mainly for the typo

File: .github/release.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
+changelog:
```
[the remark this answers is not in the document]
Comment (Guts) [reply .github/release.yml:1]: I guess it's used during the release creation. This is what I use for QDT https://github.com/Guts/qgis-deployment-cli/releases/tag/0.10.0

[the remark this answers is not in the document]
Comment (Guts) [reply .github/release.yml:1]: If not, I will modify our GH Actions later

[the remark this answers is not in the document]
Comment (Gustry) [reply .github/release.yml:1]: Not yet indeed.
https://github.com/Guts/qgis-deployment-cli/blob/main/.github/workflows/build_release.yml#L164

It's a GH Action https://github.com/softprops/action-gh-release
Which is not used in this repository : https://github.com/opengisch/qgis-plugin-ci/blob/master/.github/workflows/release.yml#L97

[the remark this answers is not in the document]
Comment (Guts) [reply .github/release.yml:1]: It's not related to Github action directly, it's a Github feature https://docs.github.com/en/repositories/releasing-projects-on-github/automatically-generated-release-notes

[the remark this answers is not in the document]
Comment (Guts) [reply .github/release.yml:1]: It's not related to Github action directly, it's a Github feature https://docs.github.com/en/repositories/releasing-projects-on-github/automatically-generated-release-notes

But it's not clean to add an unused file into a PR. So I'll remove it

[the remark this answers is not in the document]
Comment (Guts) [reply .github/release.yml]: Removed.

**Review by Gustry — APPROVED** [review]

**Pull request merged**